client/dashboard/components/notice/style.scss

```diff
@@ -0,0 +1,90 @@
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+.dashboard-notice {
+	.dashboard-notice__icon {
```

Comment:
Do we need this nesting or can we flatten this file and have the child selectors at the top-level? To limit specificity and reduce output file size.

Reply:
Ok